webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -271,11 +271,11 @@ class SubscriptionsPage extends Component {
             <div id="subscriptions-table" className="modal-container">
               {simpleContentAccess && (
                 <Alert type="info">
-                  This organization has Simple Content Access enabled.
-                  Hosts are not required to have subscriptions attached to access repositories.
+                  {__(`This organization has Simple Content Access enabled.
+                  Hosts are not required to have subscriptions attached to access repositories.`)}
                   <br />
-                  Learn more about your overall subscription usage at
-                  {' '}<a href={SUBSCRIPTION_WATCH_URL} target="_blank" rel="noreferrer">Subscription Watch</a>.
+                  {__('Learn more about your overall subscription usage at')}
```

Comment:
This is another partial that will be difficult to translate without context.

Reply:
Using dangerous sprintf here :D

Reply:
What does dangerous sprintf mean?

Reply:
just kidding, I used the `sprintf` method from core and injected it with `dangerouslySetInnerHTML` to keep the html tags